src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1767,11 +1865,15 @@ bool entitiesQuery
       // FIXME P4: Once issue #1705 is implemented, this check can be removed.
       if (qStringFilters(sco->value, filters) != true)
       {
-        if (badInputP)
+        if (badInputP)  // Bad Input reported by higher level
```

Comment:
I guess that badInputP is a pointer. In that case, I'd suggest to be more explicit:

```
if (badInpuP != NULL)
```